### DISCUSSION TITLE
Need to show Crosshair Horizontal and vertical lines on hover in ReactApp.

### DISCUSSION THREAD

**Discussion by shankygarg30** [tradingview/lightweight-charts#1030] [Offtopic]
Hi,
I implemented the charts in react app but I am not able to see the Crosshair horizontal and vertical lines on hovering the chart. I am using AreaSeries with below setting

```js
chart.applyOptions({
                crosshair: {
                    mode: CrosshairMode.Normal,
                    vertLine: {
                        color: 'FF0000',
                        width: 2,
                        visible:true,
                        labelVisible:true
                    },
                    horzLine: {
                        color: 'FF0000',
                        width: 2,
                        visible:true,
                        labelVisible:true
                    }
                }
            })
```

Can someone please tell me what I am missing here? 
When i tried this implementation in html page then it works fine. Not able to understand what might be the reason behind cross hair not working in reactapp.

**Comment by timocov** [comment]
I think it would be much better if you create a repro for your issue so anyone could check it out locally.

**Reply by shankygarg30** [reply]
Hi I checked my css and found that we had blocker pointer events on canvas elements. Now removed that and crosshair is working.
